src/ORM/DataQuery.php

```diff
@@ -1025,8 +1025,6 @@ public function applyRelation($relation, $linearOnly = false)
      * Join the given has_many relation to this query.
      * Also works with belongs_to
      *
-     * Doesn't work with polymorphic relationships
-     *
```

Comment:
This comment was already wrong - it had code here that made it work with polymorphic relationships before I came along. I'm just adding support for the new multiple-reciprocal has_one relations.